packages/teams-js/src/public/visualMedia.ts

```diff
@@ -0,0 +1,308 @@
+    if (!image.isSupported()) {
+      throw errorNotSupportedOnPlatform;
+    }
+    const permissions: DevicePermission = DevicePermission.Media;
+    return sendAndHandleSdkError('permissions.request', permissions);
+  }
+
+  /**
+   * To enable this image capability will let the app developer ask the user to get images from camera/local storage
+   * @beta
+   */
+  export namespace image {
+    /**
+     * CameraImageProperties is for the image taken from the camera
+     * @beta
+     */
+    export interface CameraImageProperties extends VisualMediaProps {
+      /**
+       * The source in CameraImageProperties should always be CameraProps
+       */
+      sourceProps: CameraProps;
+
+      /**
+       * The common additional properties for image in mobile devices
+       */
+      commonImageprops: ImageProperties;
+    }
+
+    /**
+     * CameraImageProperties is for the image taken from the camera
+     * @beta
+     */
+    export interface GalleryImageProperties extends VisualMediaProps {
+      /**
+       * The source in GalleryImageProperties should always be GalleryProps
+       */
+      sourceProps: GalleryProps;
+
+      /**
+       * The common additional properties for image in mobile devices
+       */
+      commonImageprops: ImageProperties;
+    }
+
+    /**
+     *  All properties in ImageProperties are optional and have default values in the platform
+     *  Additional properties for image in mobile devices
+     * @beta
+     */
+    export interface ImageProperties {
+      /**
+       * Optional; indicate if inking on the selected Image is allowed or not
+       * Default value is false
+       */
+      shouldAllowInkingOnImages?: boolean;
+
+      /**
+       * Optional; indicate if putting text stickers on the selected Image is allowed or not
+       * Default value is false
+       */
+      shouldAllowTextStickersOnImages?: boolean;
+
+      /**
+       * Optional; indicate if the filter mode is enabled on the selected image
+       * Default value is false
+       */
+      enableFilterOnImages?: boolean;
+
+      /**
+       * Optional; Lets the developer specify the image output formats, more than one can be specified.
+       * Default value is Image.
+       */
+      imageFormatsAllowedToBeSelected?: ImageOutputFormats[];
+    }
+
+    /**
+     * Specifies the image output formats.
+     * @beta
+     */
+    export enum ImageOutputFormats {
+      /** Outputs image.  */
+      Image = 1,
+      /** Outputs pdf. */
+      PDF = 2,
+    }
+    const errorInvalidSource = 'Invalid input source.';
+    const errorInvalidCount = 'Invalid input count: Must supply the valid image counts (limit to 10).';
+    const errorInvalidResponse = 'Invalid response: Received more images than the specified max limit in the response.';
+    /**
+     * Capture one or multiple image(s) using camera.
+     * @beta
+     * @param imageInputs - The input params to customize the image(s) to be captured
```

Comment:
>imageInputs [](http://example.com/codeflow?start=14&length=11)

This name doesn't match the parameter name

Reply:
Oh! Thanks for catching this up! I've updated both galleryImageInputs and cameraImageInputs names.